src/test/java/subway/StationAcceptanceTest.java

```diff
@@ -18,50 +20,86 @@
 @DisplayName("지하철역 관련 기능")
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
 public class StationAcceptanceTest {
+
+    private final String[] STATION_NAMES = {"강남역", "약삼역"};
```

Comment:
테스트에 사용하는 문자열을 배열로 선언했을 때 어떤 장점이 있을까요?
조금 더 직관적으로 하나씩 클래스 변수로 선언하면 어떨까요?

```suggestion
    private final String 강남역 = "강남역";
    private final String 역삼역 = "역삼역";
```

Reply:
여러개의 등록을 해야될 경우 호출해주어야 되는 코드가 중복적으로 발생하기때문에 배열로 선언해서 중복코드를 제거하는 방향이 효율적이라 판단했습니다